src/main/java/org/benetech/servicenet/service/dto/UserDTO.java

```diff
@@ -82,5 +85,10 @@ public UserDTO(UserProfile userProfile) {
                 .map(Shelter::getId)
                 .collect(Collectors.toList());
         }
+        if (userProfile.getOrganizations() != null) {
+            this.shelters = userProfile.getOrganizations().stream()
```

Comment:
why are we saving organizations in the shelters field?

Reply:
Thanks, Good catch. I missed that.